src/homework01.py

```diff
@@ -0,0 +1,327 @@
+#!/usr/bin/python3
+# -*- coding: utf-8 -*-
+# coding: utf8
+
+"""
+EPAM python q4 homework 01
+Sergey Streltsov 2019-10-18
+"""
+
+import csv
+import doctest
+import glob
+import itertools
+from itertools import groupby
+import random
+import re
+import sys
+from time import localtime
+import unittest
+import xml.etree.ElementTree as etree
+
+# 1 line: Output
+print('Hello, world!')
+
```

Comment:
it looks as a good idea to separate tasks from each other with 2 empty lines everywhere